commit 80226fcad7525a9818fcf6064e2ffa7e82ea00cb
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-09-11T17:58:28+01:00 (4 weeks ago)
    
    TOOLS-2314 eng.git GO_ENV should include GOCACHE
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/tools/mk/Makefile.go_prebuilt.defs b/tools/mk/Makefile.go_prebuilt.defs
index 353617d..3ada027 100644
--- a/tools/mk/Makefile.go_prebuilt.defs
+++ b/tools/mk/Makefile.go_prebuilt.defs
@@ -122,6 +122,7 @@ GO_ENV +=			GOPATH="$(TOP)/$(GO_GOPATH)"
 GO_ENV +=			GOARCH="$(GO_GOARCH)"
 GO_ENV +=			GOOS="$(GO_GOOS)"
 GO_ENV +=			PATH="$(TOP)/$(GO_INSTALL)/bin:$$PATH"
+GO_ENV +=			GOCACHE="$(shell realpath $(CACHE_DIR))/go_cache"
 
 #
 # The $(GO) variable should be used in place of bare invocations of "go".
